docs/UserGuide.md

```diff
@@ -411,7 +417,6 @@ _Details coming soon ..._
 
 ## Command summary
 
-_italic_ - optional
 
 | Action         | Format, Examples                                                                                                                                                                                                                                                                          |
```

Comment:
Command summary table is quite off. Might want to make adjustments! Thanks.

![image](https://user-images.githubusercontent.com/38004268/230174656-1f9b83d8-4a9f-49a4-8ef4-4c789496d3ca.png)

Reply:
Also, for `edittxn`, help to remove the italic for [STATUS ...]